### DISCUSSION TITLE
Hinzufügen von Inverter setzt Gesamtertrag zurück?

### DISCUSSION THREAD

**Discussion by Mimamau** [tbnobody/OpenDTU#1801] [General]
Ich habe einen weiteren Inverter hinzugefügt und jetzt steht mein Gesamtertrag wieder bei 0.
Bug oder Feature?

**Comment by MetaChuh** [comment]
> Bug oder Feature?

weder noch 😉

`Inverter Gesamtertrag Insgesamt` wird spätestens nach einem opendtu reboot von allen invertern die noch online sind erneut eingelesen.

`Inverter Gesamtertrag Heute` ebenfalls, solange die inverter noch online sind.
sollten einige inverter offline sein, dann sind nur diese auf null.

**Comment by Mimamau** [comment]
Ich meinte den Gesamtertrag insgesamt direkt auf dem Dashboard, über alle Inverter, der auch die Nacht und einen Reboot überlebt.

**Reply by MetaChuh** [reply]
> direkt auf dem Dashboard

bitte spezifiziere dashboard.
meinst du damit eine externes dashboard, das bei dir läuft und evtl auf +1 inverter angepasst werden muss, oder die anzeige im opendtu web interface, bzw. am opendtu display, falls vorhanden ?

(wir wissen leider nichts über dein spezifisches setup und versionen)

**Reply by Mimamau** [reply]
Ja, ich meine das OpenDTU-Webinterface.

**Reply by MetaChuh** [reply]
sind von den optionen (siehe screenshot) eines oder mehrere bei dir aktiviert ?

<img width="793" alt="Bildschirmfoto 2024-03-02 um 13 30 36" src="https://github.com/tbnobody/OpenDTU/assets/45773305/bbdcb781-c92b-4a65-8a97-b87c67532cbe">

**Reply by Mimamau** [reply]
Nein, alles aus.

**Reply by Mimamau** [reply]
![Bildschirmfoto vom 2024-03-02 15-31-23](https://github.com/tbnobody/OpenDTU/assets/23263481/2ca9afad-d2d6-4b02-acd9-b050cc37123b)
Diese Anzeige meine ich.

**Reply by MetaChuh** [reply]
ich kann es ebenfalls noch immer nicht nachvollziehen.

ich habe gerade auf opendtu v24.2.16 einen 6. inverter hinzugefügt.
egal ob mit oder ohne reboot, sehe ich den gesamtertrag und tagesertrag.
(siehe screenshot)

<img width="394" alt="Bildschirmfoto 2024-03-02 um 16 20 35" src="https://github.com/tbnobody/OpenDTU/assets/45773305/a49d6b18-05a7-4e12-9bcc-90f67bfde080">

**Reply by Mimamau** [reply]
Ich habe auch gerade einen Inverter hinzugefügt, nichts ist passiert. 
Tja, sonst habe ich nichts verändert, kann es mir nicht erklären.

**Comment by tbnobody** [comment]
> und einen Reboot überlebt.

Das ist nicht möglich! Nach einem Reboot der DTU ist alles auf 0 und wird von den Invertern neu gelesen. Die DTU selbst speichert gar nichts. das wird alles Live aus den Invertern gelesen.
